app/views/shared/_navigation.html.erb

```diff
@@ -10,8 +10,8 @@
                     <%= link_to_unless_current "Services", admin_services_path, class: "site-nav__link" do %>
                         <span class="site-nav__current" tabindex="0">Services</span>
                     <% end %>
-                    <% if @pending_count > 0 %>
-                        <div class="badge"><%= @pending_count %></div>
+                    <% if @service_counts[:all][:pending] > 0 %>
```

Comment:
Where you are getting nested values out of a hash, you can use [`dig`](https://apidock.com/ruby/Hash/dig) instead which is a bit nicer as it won't error if any of the intermediate values are missing.

So in this case you could do: `if @service_counts.dig(:all, :pending)` 😊